### ISSUE TITLE
console_scripts that wrap python inside bash to avoid excessive shebang lines are not fixed

### ISSUE THREAD

**Issue by jdratlif** [Yelp/virtualenv-tools#26]
If you have a really long path to your venv (> 128 characters by default on CentOS 7), when pip installs a package with a console script (uvicorn or email-validator for example), it will see that the path to the interpreter is too long and wrap the python script inside a bash exec script.

These scripts are not fixed by virtualenv-tools. So the scripts don't run in the new location.

Steps to reproduce:

```bash
python3 -m venv /tmp/package-virtualenv-build-9e2c8b62a9a1af7dc0596d622e33b40152801e1f91c80625ad41a9b28598/usr/share/myorg/appname/venv
/tmp/package-virtualenv-build-9e2c8b62a9a1af7dc0596d622e33b40152801e1f91c80625ad41a9b28598/usr/share/myorg/appname/venv/bin/python -m pip install -U pip
/tmp/package-virtualenv-build-9e2c8b62a9a1af7dc0596d622e33b40152801e1f91c80625ad41a9b28598/usr/share/myorg/appname/venv/bin/python -m pip install uvicorn
head -n 2 /tmp/package-virtualenv-build-9e2c8b62a9a1af7dc0596d622e33b40152801e1f91c80625ad41a9b28598/usr/share/myorg/appname/venv/bin/uvicorn
```

You will see this at the end:

```
#!/bin/sh
'''exec' /tmp/package-virtualenv-build-9e2c8b62a9a1af7dc0596d622e33b40152801e1f91c80625ad41a9b28598/usr/share/myorg/appname/venv/bin/python "$0" "$@"
```

It's somewhat dependent on pip version. Version 9 that comes with CentOS 7 doesn't do this, but recent versions (20.2.4 and 21.3.1) do.

I think virtualenv-tools isn't searching for these bash wrapped python scripts.

**Comment by jkugler** [comment]
I just got bitten by this bug as well.

**Comment by jdratlif** [comment]
The fix was merged in 3.1.0, so I think this can be closed.
